Resources/Prototypes/_Moffstation/Entities/Objects/Weapons/Guns/Rifles/rifles.yml

```
@@ -52,7 +52,7 @@
   parent: WeaponRifleLecter
   id: WeaponRifleLecterDMR
   name: Lecter DMR
-  description: "A Dedicated Marksman Rifle designed by Sunshade Armories, built on the popular Lecter platform. Features redesigned ergonomic furniture and the latest in electronic optics to excel at long range.\nFeeds from .20 rifle magazines."
+  description: "A Dedicated Marksman Rifle designed by Sunshade Armories, built on the popular Lecter platform. Features redesigned ergonomic furniture and the latest in electronic optics to excel at long range./nFeeds from .20 rifle magazines."
```

```suggestion
  description: "A Dedicated Marksman Rifle designed by Sunshade Armories, built on the popular Lecter platform. Features redesigned ergonomic furniture and the latest in electronic optics to excel at long range.\nFeeds from .20 rifle magazines."
```
`/n` is not an escape character, so you will end up with literally `/n` in the description if you do this.

---

I have no idea how I accidentally did this lol